tools/cldr-code/src/main/java/org/unicode/cldr/tool/CLDRModify.java

```diff
@@ -2102,6 +2102,45 @@ public void handlePath(String xpath) {
                 seen.addAll(paths);
             }
         });
+
+        fixList.add('U', "Un-drop inheritance", new CLDRFilter() {
+            final private String baseDir = "/Users/tbishop/Documents/WenlinDocs/Organizations/Unicode/CLDR_job/2022_10_07_pre/";
```

Comment:
One reason this is only a draft PR is that the path to the "old" files is hard coded -- this might be a one-time special operation, but if this change to CLDRModify.java really goes in, we could generalize it to make it potentially reusable

another reason to keep it draft for now is that we should study the changes very carefully due to the huge number of locales and paths affected

Reply:
2022_10_07_pre folder gets xml from 80029f1d4dc0309acaaf9e9f4d005675c75a7163 "pre-drop"

Maybe we should also take into account changes between commit 02fbf4b88bfa95f52850bca6659f0c756733759d ("post-drop") and current, if any